janis_core/workflow/workflow.py

```diff
@@ -1230,9 +1221,40 @@ def get_dot_plot(
 
     def version(self):
         meta: WorkflowMetadata = self.bind_metadata() or self.metadata
-        if meta:
+        if meta and meta.version:
             return meta.version
 
+    def apply_input_documentation(
+        self,
+        inputs: Dict[str, Union[InputDocumentation, str, Dict[str, any]]],
+        should_override=False,
+        strict=False,
```

Comment:
I think it will be useful to have docstring to briefly explain what `should_override` and `strict` options are for